src/test/java/rx/plugins/RxJavaPluginsTest.java

```diff
@@ -238,6 +246,11 @@ public Date call(Date date) {
         // just use defaults
     }
 
+    // inside test so it is stripped from Javadocs
+    public static class RxJavaSingleExecutionHookTestImpl extends RxJavaSingleExecutionHook {
```

Comment:
I get your point. However I think `mock` won't help here as I need default behavior of default hook (otherwise in each test we will need to setup mock behavior). I would like to use `spy` to make both `verify` and default behavior possible. I'll make a commit soon so you'll be able to check my proposition.